risc0/zkvm/src/exec/env.rs

```diff
@@ -63,6 +68,17 @@ pub struct ExecutorEnv<'a> {
 
 impl<'a> ExecutorEnv<'a> {
     /// Construct a [ExecutorEnvBuilder].
+    /// # Example
+    /// ```
+    /// # use risc0_zkvm::{
+    /// #   ExecutorEnv,
+    /// #   ExecutorEnvBuilder,
+    ///     serde::to_vec};
+    ///
+    /// let a: u64 = 400;
+    ///
+    /// let env = ExecutorEnv::builder().build();
```

Comment:
This example doesn't seem especially useful

Reply:
Maybe instead of an example, add a redirection to other parts of the doc, e.g. "Combine with [ExecutorEnvBuilder] functions like [...add_input] to set up the environment and I/O data your guest needs. Finalize with [ExecutorEnvBuilder::build] to get a configured [ExecutorEnv]."